lms/static/coffee/src/customwmd.coffee

```diff
@@ -126,7 +126,7 @@ if Markdown?
       $elem.empty()
       _append = appended_id || ""
       wmdInputId = "wmd-input#{_append}"
-      $wmdPreviewContainer = $("<div>").addClass("wmd-preview-container")
+      $wmdPreviewContainer = $("<div>").addClass("wmd-preview-container").attr("aria-label", "HTML preview of post")
```

Comment:
This should use ``gettext`` to be localized. I'd probably also put it on a new line.